topics/neo/index.md

```diff
@@ -0,0 +1,13 @@
+---
+created_by: Hongfei Da, Erik Zhang
+display_name: NEO
+logo: neo.png
+released: July 15, 2016
+short_description: A distributed network for the Smart Economy.
+topic: neo
+related: blockchain, cryptocurrency
+url: https://www.neo.org/
+github_url: https://github.com/neo-project
+wikipedia_url: http://docs.neo.org/
```

Comment:
Change this URL to https://en.wikipedia.org/wiki/NEO_(cryptocurrency) ?

Reply:
sure, no problem. I have updated logo and wiki url.